web/owlspa/src/actions/cleanUpTmpPatchFiles.js

```diff
@@ -19,6 +19,8 @@ const cleanUpTmpPatchFiles = (patchId) => {
       if (response.status >= 400) {
         throw new Error('error cleaning up temporary patch files, server status: ' + response.status);
```

Comment:
it would actually be great if we didn't have to send a request from the front end to tell the back end to clean up the temp patch files after successful patch creation, but it's probably worth raising in a separate issue to resolve.